storybook/storybook-react/src/Grid/Grid.docs.mdx

```diff
@@ -26,31 +32,17 @@ Daarom bestaat er een compacte variant van het grid.
 
 ### Verticale marge
 
-In tegenstelling tot de horizontale marges zijn de verticale instelbaar.
-De opties `paddingVertical`, `paddingTop` en `paddingBottom` voegen witruimte boven en/of onder het grid toe.
-Dit is handig in een vlak met een achtergrondkleur zoals [Footer](?path=/docs/react_containers-footer--docs) of
+In tegenstelling tot de horizontale marges tussen de kolommen zijn de verticale erboven en eronder instelbaar.
+De opties `paddingVertical`, `paddingTop` en `paddingBottom` voegen daar witruimte toe.
+Dit is handig in een gekleurd vlak zoals [Footer](?path=/docs/react_containers-footer--docs) of [Highlight](?path=/docs/react_containers-highlight--docs) of tussen twee opvolgende grids.
 [Spotlight](?path=/docs/react_containers-spotlight--docs) of tussen twee opvolgende grids.
```

Comment:
Copy paste error? It says 'of tussen twee opvolgende grids' twice.

Reply:
Yes, merge error after renaming Highlight to Spotlight. Fixed.